jpo-geojsonconverter/pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.5.0</version>
+        <version>2.7.6</version>
```

Comment:
Why not use the latest version compatible with Java 11 which is 2.7.8?

Reply:
I went with what was set in the conflict monitor for this one. We could try 2.7.8 and verify everything still works.